docker-compose.dev.yml

```diff
@@ -0,0 +1,79 @@
+services:
+  redis-standalone-1:
+    image: redis/redis-stack-server:7.4.0-v1
+    ports:
+      - "7001:6379"
+    volumes:
+      - ./config/redis/standalone.conf:/redis-stack.conf
+      - ./data/redis/7001:/data
+    env_file:
+      - ./config/redis/.env.redis
+  redis-standalone-2:
+    image: redis/redis-stack-server:7.4.0-v1
+    ports:
+      - "7002:6379"
+    volumes:
+      - ./config/redis/standalone.conf:/redis-stack.conf
+      - ./data/redis/7002:/data
+    env_file:
+      - ./config/redis/.env.redis
+  redis-node-01:
+    image: redis/redis-stack-server:7.4.0-v1
+    command: [ "bash", "/home/runner/mounted/node.sh", "7101" ]
+    network_mode: "host"
```

Comment:
`network: host` doesn't work by default on docker desktop for mac and windows. Maybe we should map ports explicitly for ease of first setup?

Reply:
Ok